src/GuiBase/Timeline/Configurations.h

```diff
@@ -1,43 +1,6 @@
 #ifndef RADIUMENGINE_TIMELINE_CONFIG_HPP_
 #define RADIUMENGINE_TIMELINE_CONFIG_HPP_
```

Comment:
now, guard is as simple as #pragma once.
here, the guard do not respect the guidelines. It is neither related to the name of the file nor the new guard `#pragma once`